server/configuration/base.yaml

```diff
@@ -1,5 +1,6 @@
+# Replace [[xxx]] with valid values in their corresponding fields. See side comments for an example below.
 server:
-  port: 8000
+  port: [[omsupply_port]] # e.g. 8000
```

Comment:
I think the defaults are helpful - for devs it means that you just clone the repo and you should be able to run the project. I'm guessing that you're doing a replace with setup factory, which is why the change? If so - is is possible to use regex and search for `port:[\w\W]*/n` for example?

Reply:
Yes to make easy for setup factory to replace old config details in yaml file. Updated with 82988f868